views/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Personal Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+
+<body>
+    <section class="toolbar">
+        <img src="../images/image_geometry_1.png" alt="navbar image">
+        <nav>
+            <img class="navbar-image" src="../images/ic_menu.svg" alt="navbar menu">
+        </nav>
+    </section>
+
+    <main>
+        <section class="headline ">
+            <section class="intro">
+                <h1 class="name">Hector Torres</h1>
+                <p class="description">Hello! I am a software developer! I can help you build a product, feature or
+                    website.
+                    Taka a look
+                    of my works. If you like what you see and have a project you need coded, don’t hesitate and contact
+                    me.
+                </p>
+                <button class="btn" type="button">Start collaboration</button>
```

Comment:
[OPTIONAL]
- [ ] You can add some styling to button like: Active, Hover and Pressed (see design)